modules/plugin/imagemosaic/src/main/java/org/geotools/gce/imagemosaic/RasterLayerResponse.java

```diff
@@ -804,6 +804,9 @@ private MosaicElement createMosaic() throws IOException {
                                     // Adding globalRoi to the output
                                     RenderedOp rop = (RenderedOp) mosaic;
                                     rop.setProperty("ROI", in.roi);
+
+                                    mosaic = footprintBehavior.postProcessMosaic(mosaic, in.roi,
```

Comment:
Ok, I just checked and I don't think this is an issue. In this case presumably it should only matter if we have a footprint that has holes in it right? In that case it never enters this code path because the bounds aren't rectangular (see line 759 where it converts the ROI to a rectangle). Maybe there is a case I'm not thinking of though.